pkg/crc/cluster/cluster.go

```diff
@@ -182,13 +182,14 @@ func addPullSecretToInstanceDisk(sshRunner *ssh.SSHRunner, pullSec string) error
 	return nil
 }
 
-func waitForOpenshiftAPIServer(oc oc.OcConfig) error {
+func waitForOpenshiftAPIServer(oc oc.OcConfig, resource string) error {
 	waitForApiServer := func() error {
-		_, stderr, err := oc.RunOcCommand("get", "pods")
+		stdout, stderr, err := oc.RunOcCommand("get", resource)
 		if err != nil {
 			logging.Debug(stderr)
 			return &errors.RetriableError{Err: err}
 		}
+		logging.Debug(stdout)
```

Comment:
dunno if that added debug is intentional?

Reply:
@cfergeau yes this is intentional, would help us if some resource not able to get after a given retry.